src/domain/app/i18n/fi.json

```diff
@@ -333,6 +333,11 @@
     "pageTitle": "Muokkaa paikkaa",
     "title": "Muokkaa paikkaa"
   },
+  "editPriceGroupPage": {
+    "pageDescription": "Muokkaa Linked Eventsiin tallennettua hintaryhmää.",
```

Comment:
Here too the term "asiakaryhmä" should be used in the different languages.

Reply:
Changed as suggested